python/ray/dashboard/client/src/pages/dashboard/memory/MemorySummary.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  createStyles,
+  TableCell,
+  TableRow,
+  Theme,
+  withStyles,
+  WithStyles,
+} from "@material-ui/core";
+import React from "react";
+import { MemoryTableSummary } from "../../../api";
+
+const styles = (theme: Theme) =>
+  createStyles({
+    cell: {
+      padding: theme.spacing(1),
+      textAlign: "center",
+      "&:last-child": {
+        paddingRight: theme.spacing(1),
+      },
+    },
+    expandCollapseCell: {
+      cursor: "pointer",
+    },
+    expandCollapseIcon: {
+      color: theme.palette.text.secondary,
+      fontSize: "1.5em",
+      verticalAlign: "middle",
+    },
+    extraInfo: {
+      fontFamily: "SFMono-Regular,Consolas,Liberation Mono,Menlo,monospace",
+      whiteSpace: "pre",
+    },
+  });
+
+type Props = {
+  memoryTableSummary: MemoryTableSummary;
+  initialExpanded: boolean;
+};
+
+type State = {
+  expanded: boolean;
+};
+
+class MemorySummary extends React.Component<
+  Props & WithStyles<typeof styles>,
+  State
+> {
+  state: State = {
+    expanded: this.props.initialExpanded,
+  };
+
+  toggleExpand = () => {
+    this.setState((state) => ({
+      expanded: !state.expanded,
+    }));
+  };
+
+  render() {
+    const { classes, memoryTableSummary } = this.props;
+
+    return (
+      memoryTableSummary !== null && (
+        <React.Fragment>
+          <TableRow hover>
+            <TableCell className={classes.cell} />
+            <TableCell className={classes.cell}>
+              {"Total Local Reference Count: " +
```

Comment:
I would love to see tool-tips with these describing what exactly each number means. I think it would be alright to release it at first without the tool-tips, but I do think they are eventually essential.